src/cljc/game/core/card.cljc

```diff
@@ -456,11 +457,11 @@
 ;;    characteristics, and is trashed.
 
 (defn convert-to-condition-counter
-  [{:keys [cid side title zone]}]
+  [{:keys [cid code side title zone]}]
   (map->Card
     {:cid cid
+     :code code
      :printed-title title
-     :seen true
```

Comment:
This isn't needed? We have a check for displaying all card counters in the gameboard?

Reply:
The `:seen` property is removed whenever a corp card is installed. I had the option to either add a check during install to not remove the `:seen` or on moving to the discard. Since we already have logic to add `:seen` to rezzed cards being moved into the discard I chose to just expand that existing check to include condition counters.